includes/admin/class-wcsatt-admin.php

```diff
@@ -36,7 +36,7 @@ public static function init() {
 		add_action( 'woocommerce_product_write_panels', __CLASS__ . '::product_write_panel' );
 
 		// Processes and saves the necessary post meta.
-		add_action( 'woocommerce_process_product_meta', __CLASS__ . '::process_product_meta' );
+		add_action( 'woocommerce_process_product_meta', __CLASS__ . '::process_product_meta', 15, 1 );
```

Comment:
I also made sure it saves the meta data after WooCommerce has done it's first, not before.